ci/test-helm

```diff
@@ -13,7 +13,7 @@ echo -e "hub:\n  url: $HUB_URL" > helm-chart/travis-binder.yaml
 
 echo "building helm chart"
 cd helm-chart
-./build.py --commit-range ${TRAVIS_COMMIT_RANGE}
+./build.py --commit-range ${TRAVIS_COMMIT_RANGE} --extra "${TRAVIS_REPO_SLUG}$(git log -1 --pretty=%B | head -n1 | sed 's/^.*\(#[0-9]*\).*/\1/')"
```

Comment:
I guess this magically only displays the commits that are being added?

Reply:
Well, actually as binderhub is being build for each merged PR, then it only extract the PR number. 

- `${TRAVIS_REPO_SLUG}` is `jupyterhub/binderhub`
- `log -1` logs only the latest commit
- `--pretty=%B`  only the message

As the message is `Merge PR #123 into master`, `sed 's/^.*\(#[0-9]*\).*/\1/')"` extract the number. 

Thus `extra` is only `jupyterhub/binderhub#123`.

It will of course not work if we push commits directly.